plugins/modules/repository_information.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/python
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: repository_info
+short_description: A module that returns information about GitHub repositories
+description:
+  - "A module that fetches information about repositories that a GitHub user has access to inside an organization."
+options:
+    token:
+        description:
+            - GitHub API token used to retrieve information about repositories a user has access to
+        required: true
+        type: str
+    enterprise_url:
+        description:
+            - If using a token from a GitHub Enterprise account, the user must pass an enterprise URL
+        required: false
+        type: str
+    organization_name:
+        description:
+          - The organization that the information is within the scope of.
```

Comment:
"The organization that the information is within the scope of." -> The organization in which the query will be run.